docs/source/conf.py

```diff
@@ -261,19 +261,23 @@ def process_readme():
 
 # Auto-generate documentation pages
 current_dir = os.path.abspath(os.path.dirname(__file__))
-target_dir = os.path.join(current_dir, "reference")
+reference_dir = os.path.join(current_dir, "reference")
+predefined_reference_dir = os.path.join(current_dir, "_reference")
```

Comment:
Could we have a better name than `_reference`? Perhaps `custom_reference` or something? It would make more sense that the auto-generated reference folder would be `_reference` IMO

Reply:
Ok, I renamed to `custom_reference`.